test/utils.js

```diff
@@ -206,4 +206,37 @@ export const loadSpecTestData = function (specPath, renderData) {
     }
 
     return tests;
+};
+
+export const getAccounts = function () {
```

Comment:
Thanks to @carsonfarmer for a good call on removing the hardcoded strings in the tests. This feels much cleaner and DRYs out the code too.